src/main/java/com/Setu/SetuHomePageService/Authentication/UserLogIn.java

```diff
@@ -8,7 +11,10 @@ public boolean ValidateUserCredentials(String username, String password){
 
     private boolean ValidateUserName(String username){
         // Todo: - Check the username is in the proper format.
```

Comment:
Is todo still required after writing the code?

Reply:
I will remove those comments.